src/dbally/prompts/common_validation_utils.py

```diff
@@ -47,3 +47,23 @@ def check_prompt_variables(chat: ChatFormat, variables_to_check: Set[str]) -> Ch
             "You need to format the following variables: {variables_to_check}"
         )
     return chat
+
+
+def extract_system_prompt(chat: ChatFormat) -> Tuple[ChatFormat, str]:
```

Comment:
Maybe it would be more convienent as a method on ChatFormat class?

Reply:
`ChatFormat` is not a class, it's just an type alias for tuple here.